client/src/screens/SupportTeam/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment, Component } from 'react';
+
+class SupportTeam extends Component {
+	state = {};
+	componentDidMount() {}
+	render() {
+		return (
+			<Fragment>
+				<h1>Hello from the SupportTeam</h1>
+			</Fragment>
+		);
```

Comment:
Looks like a good candidate for stateless functional component

And I think we can get rid of the Fragment wrapper too since it's just one element 👍